### PR TITLE
Add clarification how to set up comments for shipment

### PR DESCRIPTION
**Pull request by EvheniiaKolkhiieva**
## Purpose of this pull request

_Describe the goal and the type of changes this pull request covers. Tell us what changes you are making and why._

This pull request (PR) adds clarification how to set up comments for shipment

## Magento release version

_Which Magento release(s) are affected by the content changes: 2.4, 2.3? Specify a patch release number, if applicable._

## Product editions

_Is this update specific to a product edition: Magento Open Source only, Adobe Commerce only?_

- no

_Is this update specific to an installed feature extension: B2B extension, other feature set (please, specify)?_

- no

## Affected documentation pages

_List HTML links for affected pages on <https://docs.magento.com/user-guide/>._

- https://docs.magento.com/user-guide/sales/order-ship.html

whatsnew
Updated the [Shipping an Order](https://docs.magento.com/user-guide/sales/order-ship.html) topic to include steps to configuring shipping comments for email to customers.

### CODE REVIEW

File: src/sales/shipments-create.md [hunk 1]
```diff
@@ -42,6 +42,31 @@ For each line item in the order, modify the **Qty to Ship** as needed.
...
+
+- In **Shipment Comment Email Sender**, select the person from whom the shipment comment email will be sent. The default Magento 2 offers five email addresses.
+
+- In **Shipment Comment Email Template**, select the template based on your requirement or select the default option.
+
+- In **Shipment Comment Email Template for Guests**, choose the template for customers who have not created an account in your store.
+
+- In **Shipment Comment Email Copy To**, enter the multiple email addresses to whom you want to send Shipment comment email copy. Separate email addresses with a comma.
+
+- Send **Shipment Comment Email Copy Method**, select bcc(blind carbon copy) or separate email copy method based on your requirement.
```
Comment (leslietilling) [inline src/sales/shipments-create.md:63]:
```suggestion
1. Expand the **Shipment Comments** section and modify the settings as needed:

   - The **Enabled** option is set to `Yes` by default, which means that the email is sent to a customer when a shipping comment is entered.

   - For **Shipment Comment Email Sender**, select the person from whom the shipment comment email is sent. The default offers five email addresses.

   - For **Shipment Comment Email Template**, select the template based on your requirement or select the default option.

   - For **Shipment Comment Email Template for Guests**, choose the template used for customers who do not have an account in your store.

   - For **Shipment Comment Email Copy To**, enter the email addresses to send a shipment comment email copy. Separate multiple email addresses with a comma.

   - For **Shipment Comment Email Copy Method**, select `bcc` (blind carbon copy) or `separate email copy` method based on your preference.
```

File: src/sales/shipments-create.md [hunk 2]
```diff
@@ -42,6 +42,31 @@ For each line item in the order, modify the **Qty to Ship** as needed.
...
+
+1. Click <span class="btn">Save Config</span>.
+
+    ![]({% link images/images/shipment-coments.png %}){: .zoom}
+    _Shipment Coment Configuration_
```
Comment (leslietilling) [inline src/sales/shipments-create.md:68]:
```suggestion
    _Shipment comment configuration_
```

**Review by leslietilling — CHANGES_REQUESTED** [review]

**Comment by EvheniiaKolkhiieva** [comment]
Hi @leslietilling , thank you for the review. I have implemented your suggestions, could you please approve this PR?
Thank you!

File: src/sales/shipments-create.md (hunk 2 above)
Comment (EvheniiaKolkhiieva) [reply src/sales/shipments-create.md]: Done ✅

File: src/sales/shipments-create.md (hunk 1 above)
Comment (EvheniiaKolkhiieva) [reply src/sales/shipments-create.md]: Done ✅

File: src/sales/shipments-create.md [hunk 3]
```diff
@@ -42,6 +42,31 @@ For each line item in the order, modify the **Qty to Ship** as needed.
 
 1. When the shipment is ready, click <span class="btn">Submit Shipment</span>.
 
+### To set up comments for shipment do the following:
+
+1. On the _Admin_ sidebar, go to **Stores** > **Settings** > **Configuration**.
```
[the remark this answers is not in the document]
Comment (EvheniiaKolkhiieva) [reply src/sales/shipments-create.md]: Done ✅

File: src/sales/shipments-create.md [hunk 4]
```diff
@@ -42,6 +42,31 @@ For each line item in the order, modify the **Qty to Ship** as needed.
 
 1. When the shipment is ready, click <span class="btn">Submit Shipment</span>.
 
+### To set up comments for shipment do the following:
```
[the remark this answers is not in the document]
Comment (EvheniiaKolkhiieva) [reply src/sales/shipments-create.md]: Done ✅

**Review by leslietilling — APPROVED** [review]

**Comment by leslietilling** [comment]
@magento import pr to magento-devdocs/merchdocs

**Comment by leslietilling** [comment]
@magento import pr to magento-devdocs/merchdocs

**Comment by magento-engcom-team** [comment]
@leslietilling the pull request successfully imported.

**Pull request merged**